assets/styles/styles.css

```
@@ -0,0 +1,663 @@
+}
+
+.nav_phone:hover .nav_tooltip {
+  visibility: visible;
+}
+.nav_phone .nav_tooltip::after {
+  content: " ";
+  position: absolute;
+  top: 50%;
+  left: 100%; /* To the right of the tooltip */
+  margin-top: -5px;
+  border-width: 5px;
+  border-style: solid;
+  border-color: transparent transparent transparent black;
+}
+/*-----end phone tooltip-----*/
+
+.nav_menu-toggle {
+  display: block;
+  position: relative;
+  -webkit-user-select: none;
+  user-select: none;
+  background: none;
+  border: 0 transparent;
+  outline: none;
+}
+
+.nav_menu-toggle a {
+  text-decoration: none;
+  color: #232323;
+  transition: color 0.3s ease;
+}
+
+.nav_menu-toggle span {
+  display: block;
+  width: 33px;
+  height: 4px;
+  margin-bottom: 5px;
+  position: relative;
+  background: #00343D;
+  border-radius: 3px;
+  z-index: 2;
+  transform-origin: 4px 0px;
+  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+  background 0.5s cubic-bezier(0.77,0.2,0.05,1.0),
+  opacity 0.55s ease;
+}
+
+.nav_menu-toggle span:first-child {
+  transform-origin: 0% 0%;
+}
+
+.nav_menu-toggle span:nth-last-child(2) {
+  transform-origin: 0% 100%;
+}
+
+.nav_menu-toggle.open span {
+  opacity: 1;
+  transform: rotate(45deg) translate(-10px, -10px);
+  background: #00D8A0;
+}
+
+.nav_menu-toggle.open span:nth-last-child(3) {
+  opacity: 0;
+  transform: rotate(0deg) scale(0.2, 0.2);
+}
+
+.nav_menu-toggle.open span:nth-last-child(2) {
+  transform: rotate(-45deg) translate(-10px, 10px);
+}
+
+.nav_menu {
+  position: absolute;
+  width: 100%;
+  margin: -26px 0 0 -28px;
+  padding: 100px 0 0 0;
+  height: calc(100% - 100px);
+  background: #fff;
+  list-style-type: none;
+  -webkit-font-smoothing: antialiased;
+  transform-origin: 0% 0%;
+  transform: translate(-100%, 0);
+  font-family: 'GTWalsheim','';
+  transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0);
+  list-style-type: none;
+  z-index: 1;
+}
+
+.nav_menu li {
+  position: relative;
+  padding: 10px 0;
+  font-size: 1.2em;
+  text-align: center;
+  font-weight: bold;
+}
+
+
+
+.nav_menu li a,
+.nav_phone {
+  text-decoration: none;
+  color: #00343D;
+
+}
+.nav_menu li a:hover + span,
+.nav_menu li a:active + span {
+  border-radius: 100%;
+  display: block;
+  background: black;
+  width: 8px;
+  height: 8px;
+  position: absolute;
+  bottom: 0;
+  left: calc(50% - 8px);
+}
+
+
+.nav_menu .contactLink {
+  margin-top: 50px;
+}
+
+.nav_menu .contactLink a {
+  font-weight: bold;
+  background: #0BD8A2;
+  border-radius: 50px;
+  padding: 10px 25px;
+}
+
+.nav_menu.open {
+  transform: none;
+}
+
+/*-------------fin seccion de NAV --------------------*/
+
+.section1 {
+  text-align: center;
+  margin-top:70px;
+}
+
+.section1_title {
+  margin: 0;
+  font-size: 2em;
+  line-height: 106%;
+  letter-spacing: -0.533333px;
+  color: #00343D;
+}
+
+.secondary-title{
+  font-weight: bold;
+  font-size: 2em;
+  line-height: 106%;
+  letter-spacing: -0.533333px;
+  color: #00343D;}
+
+.third-title{
```

Until reaching this line you were using a space before the bracket. What happened @edwartexe ? 😄 
```suggestion
.third-title {
```